README.md

```diff
@@ -16,6 +16,34 @@ For details about configuration and deployment of RBD and CephFS CSI plugins, se
 
 For example usage of RBD and CephFS CSI plugins, see examples in `examples/`.
 
+## Support Matrix
+
+Here is the matrix that describes the features supported by the different
+versions of Ceph CSI driver
+
+|              |        Features             | CSI driver Version | Kubernetes Version   |
+|--------------|-----------------------------|--------------------|----------------------|
+| **CephFS**   | Snapshot                    |         1.0        |       >=1.13         |
```

Comment:
need to updated driver version to `1.0.0` and `v0.3.0`

Reply:
Cephfs, does not support snapshot.., hence will remove this